CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Features
+- Update Deno config to allow passing `--location` runtime flag.
```

Comment:
Could you put this under "build tool" please. Changes are categorised by what area of the codebase they are related to.

Reply:
Good call. Will update now 👍🏻